build/tools.mk

```diff
@@ -83,3 +83,34 @@ PATCH   = patch
 GENASSYM = $(TOPDIR)/sys/script/genassym.sh
 YACC	= byacc
 GZIP	= gzip -9
+
+dump-tools:
+	echo "CC       ${CC}"
```

Comment:
Wouldn't it be easier if you just dumped everything in following format: `CC=${CC}` ?
Then you could use `source` command from bash to include all variables.

Reply:
Right, it is better.